recipes/cogwheel-pe/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "cogwheel-pe" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cogwheel_pe-{{ version }}.tar.gz
+  sha256: e728483d6a701b8d0035f120c3541c847e351148038d49951475e32dd5dfafdf
+
+build:
+  skip: true  # [win]
```

Comment:
It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

Reply:
Thanks! I reverted the package to noarch, failing on Windows.

Reply:
It is due to a missing dependency, so it is fine. This package will be ready when the dependency is available.